src/docbuild/cli/cmd_validate/__init__.py

```diff
@@ -19,15 +19,26 @@
     nargs=-1,
     type=click.Path(exists=True, dir_okay=False, path_type=Path),
 )
+@click.option(
+    '--validation-method',
+    type=click.Choice(['jing', 'lxml'], case_sensitive=False),
+    default='jing',
+    help="Choose validation method: 'jing' (default) or 'lxml'",
```

Comment:
The default value is already shown when you run `docbuild validate --help`. We can remove this:


```suggestion
    help="Choose validation method: 'jing' or 'lxml'",
```

This is how it looks with your code:

```
docbuild validate -h
Usage: docbuild validate [OPTIONS] [XMLFILES]...

  CLI interface to validate XML configuration files.

Options:
  --validation-method [jing|lxml]
                                  Choose validation method: 'jing' (default)
                                  or 'lxml'  [default: jing]
  -h, --help                      Show this message and exit.
```